### PR TITLE
Replaces JEM with JIMM

### PR DESCRIPTION
**Pull request by jcsackett**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/154/
Test PASSed.

File: theblues/jimm.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def fetch_macaroon(self):
             log.error(message)
             return None
         except Exception as e:
-            log.info('Unable to contact jem due to: {}'.format(e))
+            log.info('Unable to contact jimm due to: {}'.format(e))
```
Comment (urosj) [inline theblues/jimm.py]: s/jimm/JIMM

File: README.rst [hunk 2]
```diff
@@ -13,7 +13,7 @@ The easiest way to install theblues is via pip::
 Parts of theblues require the use of macaroons (e.g. parsing the return of
 `Charmstore.fetch_macaroon` or interacting with `IdentityManager.discharge`).
 
-To use authenticated aspects of theblues, e.g. jem, you'll need to be able to
+To use authenticated aspects of theblues, e.g. jimm, you'll need to be able to
```
Comment (urosj) [inline README.rst]: s/jimm/JIMM

**Review by urosj — APPROVED** [review]
LGTM with small comments

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/155/
Test PASSed.

**Comment by jcsackett** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/theblues-lib-merge

**Pull request merged**